### DISCUSSION TITLE
Can I use Rich to render HTML tracebacks?

### DISCUSSION THREAD

**Discussion by dhirschfeld** [Textualize/rich#1409] [Q&A] [answered]
I need a nice HTML repr of exceptions and I've heard good things about Rich so I'm wondering if I can use Rich for this job?

I *think* the answer is yes. From poking about I think it may be possible to use the jupyter support to do this but, not being familiar with Rich itself, I'm not 100% sure so just looking for confirmation... and maybe an example! 😁

**Comment by willmcgugan** [comment, marked as answer]
Something along these lines?

```python
from rich.console import Console

console = Console(record=True)

try:
    1 / 0
except:
    console.print_exception()

console.save_html("exc.html")
```

**Reply by dhirschfeld** [reply]
Yep, perfect - thanks! :+1:
